packages/components/Atoms/Icon/TextIcon.vue

```diff
@@ -63,6 +76,9 @@ export default {
         size: this.size,
         class: `rpl-text-icon--${this.placement}`
       }
+    },
+    textIsAbsoluteUrl: function () {
```

Comment:
if `isAbsoluteUrl` is a function, no need to wrap in another function. 
```
<template>
<span v-if="isAbsoluteUrl(text)">something</span> 
</template>
<script>
methods: {
  isAbsoluteUrl,
}
</script>
```

Reply:
I spoke to Jess and she was ok with the text wrapping to the next line.

Thanks @dylankelly and @rubyllamadora

Reply:
@mayurngondhkar have another look at my comment - im talking about how you are adding this function into the vue component

Reply:
ok i see you have removed this 👍